### PR TITLE
There is no distinct add admins section in create new event page instead you can add any member from the org. The separation between admins and members was pointless and was adding redundancy so removed it.

### PR DESCRIPTION
**Pull request by Kushalrock**
What kind of change does this PR introduce?
Bug Fixes and unnecessary code removal

Did you add tests for your changes?
Tests updated to reflect the changes

Issue Number:
Closes https://github.com/PalisadoesFoundation/talawa/issues/1600

Have you read the [contributing guide]
Yes

Summary
As administrative functions will not be handled by the app the section Select Admins was no longer valid in creating events and several lists associated with it. Also, the tests that are used to check the select admin's feature were removed as they were no longer needed. 

**Screenshots supporting my PR**
Screenshot (425)
Screenshot (426)
Screenshot (427)

As can be seen from the above images out of the two members one was the admin of the organisation and one was a normal member. Both can be added to the events but there is no distinct set of options for setting up admins of the events.

### CODE REVIEW

**Comment by Kushalrock** [comment]
@palisadoes My previous PR was documented poorly sorry about that but yes the tests that I have removed were related to the select admins feature and one test that I commented out was not working anyways and was polluting the test codebase by saying it tests member and admins deletion.

**Comment by Cioppolo14** [comment]
@Kushalrock Please refer to the PR_GUIDELINES.md document in the root folder of the repo. It tells you how to automatically close your issue when the PR is merged.  Also, please fix any failed tests.

**Comment by Kushalrock** [comment]
> @Kushalrock Please refer to the PR_GUIDELINES.md document in the root folder of the repo. It tells you how to automatically close your issue when the PR is merged.  Also, please fix any failed tests. 

Thank you again. But yes I have already linked this to an issue and that issue will get close after merge is completed. Can you ask someone for review request?

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by Kushalrock** [comment]
@palisadoes The failing tests are related to admins section as it is no longer there those tests will fail😅. I commented out those test in my PR.

**Comment by palisadoes** [comment]
1. Please fix the failing tests to reflect the new desired outcome.
1. The tests will need to be appropriately changed.

**Comment by Kushalrock** [comment]
@palisadoes I have removed tests that were checking add admins section part of the code and modified tests such that now it only checks for add members section. Is this okay??

**Comment by palisadoes** [comment]
Yes, however our code coverage GitHub action now requires all submitted code to have the same or better coverage levels than the repository as a whole.

Please update the tests for the submitted files so that they meet as close to 100% test code coverage

**Comment by Kushalrock** [comment]
@palisadoes How does it check coverage? Is it decreasing coverage value perhaps because add admins test code were commented and/or removed??

**Comment by palisadoes** [comment]
If you have doubts, please paste the code coverage reports for each edited file from your system into a comments. We need to maintain the level.

**Comment by Kushalrock** [comment]
@palisadoes So should I setup codecov on my forked repo??

**Comment by palisadoes** [comment]
The CONTRIBUTING.md file should show you how to handle this

**Comment by Kushalrock** [comment]
@palisadoes The latest code coverage report.
image

**Comment by Kushalrock** [comment]
@palisadoes Any comments or things that I have to correct out in here?

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1. The overall code coverage drops below the target threshold of the repository
1. Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained.

**Comment by Kushalrock** [comment]
@palisadoes I have fixed all the tests that have come to my knowledge. Therefore, there is no delta negative change in codecov report as well. Like it was showing before my last commit that create event page had a negative delta of 3.85 %.

**Comment by palisadoes** [comment]
1. It may be happening because you changed the bottom sheet. 
1. The CodeCov service isn't 100% reliable, but to be sure, please add more comprehensive testing.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Fix the coverage

**Comment by Kushalrock** [comment]
@palisadoes I don't know why these conflicts arose. Can you look into it? I fixed the test coverage for add members' bottom sheet.

image

**Comment by palisadoes** [comment]
Please fix the conflicting file. There may also be failed tests after you do so.  

We have a new check in place. All Talawa mobile app methods, classes and functions need to have suitable comments. Please refer to this issue as  reference for what is now required. The aim is to help us improve the quality of our automated documentation on docs.talawa.io.
 
- https://github.com/PalisadoesFoundation/talawa/issues/1479

To fix this issue:

1. You will be required to remove a line containing `ignore_for_file`
1. After doing so you'll be requested to update the comments appropriately

We have also tried to make the documentation step automatically update your IDE with a custom Linter. If you have any comments, please contact @literalEval who worked on this code.

**Comment by CyberWake** [comment]
@Kushalrock  please update the following files in order to get the workflow run subsequent steps
Screenshot 2023-03-09 at 6 08 02 PM

**Comment by Kushalrock** [comment]
@CyberWake There are many files in here that I have not coded. I have used a few lines of code from a few but am not exactly an author of them. Don't you think it should be a completely another issue to do this stuff.

**Comment by literalEval** [comment]
@palisadoes should we exclude files like `test_helpers.dart` and `test_helpers_mocks.dart` from the `check_ignore.py` script ?

**Comment by Kushalrock** [comment]
@palisadoes @CyberWake One of the main reasons for the merge conflict was that someone changed the same lines tat I was editing. I had to redo some work because of that too. The more days this pr remains open, there will be many more such merge conflicts as new PRskeep getting pushed. If I have done something wrong please review and tell me I will try to right the wrong. Otherwise it would be much better if this PR gets merged and closed.

**Comment by palisadoes** [comment]
> @palisadoes should we exclude files like `test_helpers.dart` and `test_helpers_mocks.dart` from the `check_ignore.py` script ?

@CyberWake @tonythegr8 @noman2002 @DMills27 Do you think we should exclude the `test/` directory from the 300 lines of code test, or should there be specific subdirectories and files in` test/` that should be excluded? We implemented this check to make sure the code was more readable and modular.

**Comment by literalEval** [comment]
> > @palisadoes should we exclude files like `test_helpers.dart` and `test_helpers_mocks.dart` from the `check_ignore.py` script ?
> 
> @CyberWake @tonythegr8 @noman2002 @DMills27 Do you think we should exclude the `test/` directory from the 300 lines of code test, or should there be specific subdirectories and files in` test/` that should be excluded? We implemented this check to make sure the code was more readable and modular.

@palisadoes sorry to interrupt. I specified the file name without the function. I was actually talking about the script that checks for presence of `// ignore:...` directive in the files so that we can check for documentation.

**Comment by palisadoes** [comment]
> > > @palisadoes should we exclude files like `test_helpers.dart` and `test_helpers_mocks.dart` from the `check_ignore.py` script ?
> > 
> > 
> > @CyberWake @tonythegr8 @noman2002 @DMills27 Do you think we should exclude the `test/` directory from the 300 lines of code test, or should there be specific subdirectories and files in` test/` that should be excluded? We implemented this check to make sure the code was more readable and modular.
> 
> @palisadoes sorry to interrupt. I specified the file name without the function. I was actually talking about the script that checks for presence of `// ignore:...` directive in the files so that we can check for documentation.

Thanks for the clarification. Yes that would be good.

**Comment by palisadoes** [comment]
In that case @Kushalrock please fix the failing tests where the source code files are too big for the reasons stated above.

**Comment by Kushalrock** [comment]
@palisadoes Sorry I didn't get you?

**Comment by palisadoes** [comment]
Please fix any failing tests

**Review by noman2002 — CHANGES_REQUESTED** [review]
Fix the failing test

**Comment by Kushalrock** [comment]
@palisadoes I have fixed all the tests too

File: test/views/after_auth_screens/events/create_event_page_test.dart [hunk 1]
```diff
@@ -354,52 +354,52 @@ void main() {
...
-    // orgMembersList.forEach((orgMember) {
-    //   if (isAdmin) {
-    //     _adminCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    //   } else {
-    //     _memberCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    //   }
-    //   _memberCheckedMap.putIfAbsent(orgMember.id!, () => false);
-    // });
-  });
+  // testWidgets("Checking tap Inkwell work for admin list", (tester) async {
```
Comment (CyberWake) [inline test/views/after_auth_screens/events/create_event_page_test.dart:357]: This is not appreciated @Kushalrock please don't comment out tests please try fixing them.

File: test/widget_tests/after_auth_screens/events/create_event_page_test.dart [hunk 2]
```diff
@@ -403,31 +403,6 @@ void main() {
           false,
         );
       });
-      testWidgets("Testing Add Admins section", (tester) async {
```
Comment (CyberWake) [inline test/widget_tests/after_auth_screens/events/create_event_page_test.dart:406]: @Kushalrock please do not remove an existing test. Done deliberately please ignore with a clarification.

**Review by CyberWake — CHANGES_REQUESTED** [review]

File: test/views/after_auth_screens/events/create_event_page_test.dart (hunk 1 above)
Comment (Kushalrock) [reply test/views/after_auth_screens/events/create_event_page_test.dart:357]: @CyberWake That is the test for admins part of the screen which has been removed.

File: test/views/after_auth_screens/events/create_event_page_test.dart [hunk 3]
```diff
@@ -354,52 +354,52 @@ void main() {
     expect((tester.widgetList(switches).toList()[1] as Switch).value, false);
   });
 
-  testWidgets("Checking tap Inkwell work for admin list", (tester) async {
```
Comment (AdityaBirangal) [inline test/views/after_auth_screens/events/create_event_page_test.dart:357]: @Kushalrock Can you write similar test case (Inkwell work) for members list (if any) ?

**Review by AdityaBirangal — CHANGES_REQUESTED** [review]

File: test/views/after_auth_screens/events/create_event_page_test.dart (hunk 3 above)
Comment (Kushalrock) [reply test/views/after_auth_screens/events/create_event_page_test.dart:357]: It was already written and works fine

File: test/widget_tests/after_auth_screens/events/create_event_page_test.dart (hunk 2 above)
Comment (Kushalrock) [reply test/widget_tests/after_auth_screens/events/create_event_page_test.dart:406]: That is the test for admins part of the screen which has been removed.

**Comment by Kushalrock** [comment]
@palisadoes Any update regarding this??

**Review by palisadoes — APPROVED** [review]

**Pull request merged**